json-schema/response.yaml

```diff
@@ -621,41 +621,43 @@ definitions:
       datacenter:
         type: string
       slots:
-        type: object
-        additionalProperties: false
-        patternProperties:
-          ^[0-9]+$:
-            type: object
-            additionalProperties: false
-            required:
-              - alias
-              - id
-              - name
-              - size
-              - vendor
-              - occupant
-            properties:
-              alias:
-                type: string
-                description: Hardware product alias
-              id:
-                type: string
-                format: uuid
-                description: Hardware product ID
-              name:
-                type: string
-                description: Hardware product name
-              size:
-                type: integer
-                description: Number of RUs for slot
-              vendor:
-                type: string
-                description: Hardware product vendor
-              occupant:
-                description: Device assigned to this slot or null
-                anyOf:
-                  - $ref: "#/definitions/Device"
-                  - type: null
+        type: array
```

Comment:
Please create a ticket in Conch-UI for this. It'll ultimately make life easier I think but right now it'll break everything all to hell.

Reply:
ok, as long as you're ok with resolving the breakage for 2.19!